changelog.txt

```diff
@@ -1,4 +1,5 @@
 0.83.1
+ * BREAKING CHANGE: add(int,RealmOject), add(RealmObject), set(int,RealmObject), move(int,int), clear(), remove(int), remove(int), remove(RealmObject), removeAll(Collection), get(int), first(), last(), size(), where() in RealmList now throws IllegalStateException if container object has been removed or Realm instance has been closed.
```

Comment:
IMO it is not a breaking change if it before would cause a native crash. For those methods that would throw another exception say what exception that was. then this change doesn't sound so dangerous :)